app/Controller/AttributesController.php

```diff
@@ -1399,11 +1400,141 @@ public function deleteSelected($id = false, $hard = false)
             throw new MethodNotAllowedException(__('This method can only be accessed via AJAX.'));
         }
+        if (!$this->request->is('post')) {
+            throw new MethodNotAllowedException();
+        }
+        if (!isset($id)) {
+            throw new MethodNotAllowedException(__('No event ID provided.'));
+        }
+        $event = $this->Attribute->Event->fetchSimpleEvent($this->Auth->user(), $id, $params = array(
+            'fields' => array('id', 'orgc_id', 'org_id', 'user_id', 'published', 'timestamp', 'info', 'uuid')
+        ));
+        if (!$this->_isSiteAdmin()) {
+            if ($event['Event']['orgc_id'] != $this->Auth->user('org_id') || (!$this->userRole['perm_modify_org'] && !($this->userRole['perm_modify'] && $event['user_id'] == $this->Auth->user('id')))) {
+                throw new MethodNotAllowedException(__('You are not authorized to edit this event.'));
+            }
+        }
+        $selectedAttributeIds = $this->Attribute->jsonDecode($this->request->data['selected_ids']);
+
+        // tags to remove
+        $tags = $this->Attribute->AttributeTag->getAttributesTags($this->Auth->user(), $id, $selectedAttributeIds);
+        $tagItemsRemove = array();
+        foreach ($tags as $k => $tag) {
+            $tagName = $tag['name'];
+            $tagItemsRemove[] = array(
+                'name' => $tagName,
+                'value' => $tag['id'],
+                'template' => array(
+                    'name' => array(
+                        'name' => $tagName,
+                        'label' => array(
+                            'background' => isset($tag['colour']) ? $tag['colour'] : '#ffffff'
+                        )
+                    ),
+                )
+            );
+        }
+        unset($tags);
+
+        // clusters to remove
+        $clusters = $this->Attribute->AttributeTag->getAttributesClusters($this->Auth->user(), $id, $selectedAttributeIds);
+        $clusterItemsRemove = array();
+        foreach ($clusters as $k => $cluster) {
+            $name = $cluster['value'];
+            $optionName = $cluster['value'];
+            $synom = $cluster['synonyms_string'] !== '' ? ' (' . $cluster['synonyms_string'] . ')' : '';
+            $optionName .= $synom;
+
+            $temp = array(
+                'name' => $optionName,
+                'value' => $cluster['id'],
+                'template' => array(
+                    'name' => $name,
+                    'infoExtra' => $cluster['description']
+                )
+            );
+            if ($cluster['synonyms_string'] !== '') {
+                $temp['infoContextual'] = __('Synonyms: ') . $cluster['synonyms_string'];
+            }
+            $clusterItemsRemove[] = $temp;
+        }
+        unset($clusters);
+        $conditions = array();
+        if (!$this->_isSiteAdmin()) {
+            $conditions = array('Tag.org_id' => array(0, $this->Auth->user('org_id')));
```

Comment:
It should be `$conditions[]`

Reply:
Yeah it's a dumb cut and paste from the previous `editSelected`. Nice catch